core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1714,6 +1715,19 @@ public ServerBuilder setHeaders(
      * @see RequestContext#id()
      */
     public ServerBuilder requestIdGenerator(Supplier<? extends RequestId> requestIdGenerator) {
```

Comment:
Should we add `@Deprecate` and `@deprecate` Javadoc?

Reply:
Sure! I modify it.